cobalt/tools/update_required_branch_checks.py

```diff
@@ -69,11 +65,13 @@ def get_protected_branches() -> List[str]:
   branches = ['main']
   for i in range(MINIMUM_LTS_RELEASE, LATEST_LTS_RELEASE + 1)[:-1]:
     branches.append(f'{i}.lts.1+')
+  branches.append('26.android')
```

Comment:
nit: Is there a reason you don't want to just include this in the declaration of the list?